src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    background-image: url(../images/star.svg);
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
The modifier class name should follow the BEM naming convention. Instead of `&--4`, it should be named `&--4 &__star` to correctly represent its purpose and ensure that the styles are applied only to the star elements within the `stars--4` modifier.